contrib/pkginstaller/README.md

```diff
@@ -5,6 +5,9 @@ $ make ARCH=<amd64 | aarch64> NO_CODESIGN=1 pkginstaller
 
 # or to create signed pkg
 $ make ARCH=<amd64 | aarch64> CODESIGN_IDENTITY=<ID> PRODUCTSIGN_IDENTITY=<ID> pkginstaller
+
+# or to prepare a signed and notarized pkg for release
+$ make ARCH=<amd64 | aarch64> CODESIGN_IDENTITY=<ID> PRODUCTSIGN_IDENTITY=<ID> NOTARIZE_USERNAME=<appleID> NOTARIZE_PASSWORD=<appleID-password> notarize
```

Comment:
Looking at my notes for how I usually sign and notarize, I only have a DevAppID, an Username, and a AppleID password. I assume the `NOTARIZE_USERNAME` is my username, and the `NOTARIZE_PASSWORD` is the apple ID product password (looks like xxxx-xxxx-xxxx-xxxx). I also think the DevAppID is`CODESIGN_IDENTITY`. What would be the `PRODUCTSIGN_IDENTITY`?

Reply:
can you try to use the DevAppID also for the `PRODUCTSIGN_IDENTITY` i think DevAppID can be used for it if it supports signing installers as well

if the DevAppID doesn't support signing installers then you have to download new certs with installer signing capabilities from the apple developer portal

(https://www.hexnode.com/mobile-device-management/help/how-to-sign-macos-pkg-files-for-deployment-with-hexnode-mdm/)

Reply:
Tested it, looks like I need a installer signing identity as well. Going to see if I can generate one today and try it out again.